src/lab_sim/objectives/take_picture_with_wrist_camera.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<root BTCPP_format="4" main_tree_to_execute="Take wrist camera picture">
```

Comment:
```suggestion
<root BTCPP_format="4" main_tree_to_execute="Take Wrist Camera Image">
```

As far as I can tell, we capitalize the Objective names this way?

Reply:
Making this change would also mean you have to change the other Objectives that use this Objective so that the names match.

Reply:
Also, the category should probably be `Perception - 2D Image`